src/react/components/_academics/AcademicSummary/PrimarySection.js

```diff
@@ -61,7 +77,34 @@ const PrimarySection = ({ section, canViewGrades, showPoints }) => {
   );
 };
 
+const PrimarySection = ({ section, canViewGrades, showPoints }) => {
+  if (section.class.multiplePrimaries) {
+    return section.class.sections.map((sek, index) => (
+      <SingleSection
+        key={index}
+        showPoints={showPoints}
+        canViewGrades={canViewGrades}
+        klass={section.class}
+        requirementsDesignation={section.requirementsDesignation}
+        units={sek.units}
+        lawUnits={section.lawUnits}
+        grading={sek.grading}
+        sectionLabel={sek.section_label} />
+    ));
+  }
+  return <SingleSection
```

Comment:
@nkutub - We can't simply have it wrap a single section in an array?

Reply:
I thought of that, but I was not confident in the data model for sourcing all classes at the session level. I wanted to limit any variability and edge cases to only multiple primaries and source the rest of the classes from the parent node as it has always been. Basically, I wanted to reduce the risk at the cost of less DRY code.